<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Patient</class>
 <widget class="QFrame" name="Patient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>424</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>421</width>
     <height>364</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="grid_layout">
    <property name="margin">
     <number>10</number>
    </property>
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>ID:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>tx_id</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Weight  (kg):</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>tx_weight</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="tx_weight">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QComboBox" name="cb_sex">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>Female</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Male</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Other</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Name:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>tx_name</cstring>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="6">
     <widget class="QPlainTextEdit" name="tx_notes">
      <property name="maximumSize">
       <size>
        <width>50000</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QLineEdit" name="tx_age">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="4">
     <widget class="QLineEdit" name="tx_name">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Birth date:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>dt_birth_date</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="3">
     <widget class="QLineEdit" name="tx_id">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Age:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>tx_age</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QDateEdit" name="dt_birth_date">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="displayFormat">
       <string>dd/MM/yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>sex:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>cb_sex</cstring>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="act_edit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/database/resources/edit.svg</normaloff>:/database/resources/edit.svg</iconset>
   </property>
   <property name="text">
    <string>Edit/Update</string>
   </property>
   <property name="toolTip">
    <string>Edit/Update record</string>
   </property>
  </action>
  <action name="act_undo">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/database/resources/undo.svg</normaloff>:/database/resources/undo.svg</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo edition and lock record</string>
   </property>
  </action>
  <action name="act_notes">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/database/resources/notas.svg</normaloff>:/database/resources/notas.svg</iconset>
   </property>
   <property name="text">
    <string>Notes</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tx_id</tabstop>
  <tabstop>tx_name</tabstop>
  <tabstop>dt_birth_date</tabstop>
  <tabstop>tx_age</tabstop>
  <tabstop>tx_weight</tabstop>
  <tabstop>cb_sex</tabstop>
  <tabstop>tx_notes</tabstop>
 </tabstops>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
